### ISSUE TITLE
Can this tool run side files?

### ISSUE THREAD

**Issue by 621625** [dsilabs/siderunner#2]
Hi,

I'm BM, a very nice and polite guy. Can this tool run .side files?

Assuming the directory is /home/seleniumtests and the name of the file is seleniumtest.side, how would the path lines in the script look?

**Comment by hlainchb** [comment]
Hi BM,
I have not run it on .side files.  It was written to work with an older version of Selenium IDE that saved it's data in XML files.  I wouldn't be surprised if the new .side files are similar but I haven't tried that.

Basically, this library allowed us to record our tests using Selenium IDE and then run then as part of our automated test suite using nose.

**Comment by 621625** [comment]
If you have time some time please check if it can run .side files. I tried already but I could not get the syntax right.

An example with a .side file would be good, I couldn't get the path right I think or it just couldn't  handle .side files. But I didn't get an error message so I'm not sure which one it was.

**Comment by 621625** [comment]
I really, really don't like javascript so a Python tool would be useful.

**Comment by 621625** [comment]
Oh, by the way, this is a little bit funny. You haven't run side files with siderunner : )

-------- Original Message --------

**Comment by hlainchb** [comment]
Yeah, I wrote this over 10 years ago before .side files were a thing.  It appears that side files are JSON format whereas this tool read the older XML format.  I also notice that it works with Chrome now.  I stopped developing this tool because Selenium IDE was going through some huge changes and it wasn't clear to me that there was a path forward.  Happy to see it's still going strong and has some new features. I will look into porting SIDErunner to make it work with the new version.

**Comment by 621625** [comment]
The selenium record extension seems to work quite well on Chromium. That's why I want to use it (Selenium IDE)

But they require another version of Node and anyway I prefer Python tools and Python everything and Bash.

Following, in case you make an update please let me know, I can test it because I test a lot of apps. Basic testing only.

Word-of-the-day

#RoniFugate

-------- Original Message --------
